conda_smithy/templates/build_steps.sh.tmpl

```diff
@@ -29,11 +31,13 @@ CONDARC
 {%- if build_with_mambabuild %}
 GET_BOA=boa
 BUILD_CMD=mambabuild
+CONDA_INSTALL_CMD=mamba
 {%- else %}
 BUILD_CMD=build
+CONDA_INSTALL_CMD=conda
 {%- endif %}
 
-conda install --yes --quiet "{{ remote_ci_setup }}" conda-build pip ${GET_BOA:-} -c conda-forge
+${CONDA_INSTALL_CMD} install --yes --quiet "{{ remote_ci_setup }}" conda-build pip ${GET_BOA:-} -c conda-forge
```

Comment:
I'd rather we use `mamba` everywhere.

Reply:
After running into a few issues in different feedstocks where we had to switch back to conda-build, would prefer that we keep this configurable

Reply:
How is that relevant here? This command gives the same solve across feedstocks.

Reply:
Right and if some combination of packages get released that mamba can't solve, all feedstocks are stuck here. Then we have to scramble to fix it (as opposed to having another release valve to take off the pressure a bit)

Reply:
i live keeping it configurable, granted in the near future these commands will essentially be identical